TASK/TaskFollowPointRoute.md

```diff
@@ -5,8 +5,15 @@ ns: TASK
 
 ```c
 // 0x595583281858626E 0xB837C816
-void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int unknown);
+void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int mode);
 ```
+### Flags
```

Comment:
This should be switched to a C enum

```c
enum eFollowPointRouteMode {
	TICKET_SINGLE = 0,
	TICKET_RETURN = 1,
	TICKET_SEASON = 2,
	TICKET_LOOP = 3
}
```

Reply:
Roger that! I was juuuust making the change.  Would something like this work:
````c // enum FOLLOWPOINTROUTEMODE { 
    TICKET_SINGLE=0,    //Stop at the end of the route 
    TICKET_RETURN=1,      //Go once then return back once following the route backwards 
    TICKET_SEASON=2,      //As above but for forever 
    TICKET_LOOP=3         //Treat the points as a loop, i.e follow it to the end then directly back to the first position, forever. 
};`

Or should I add 'endenum?'  Sorry - I'm mostly unfamiliar with c++ syntax

Reply:
Geez that was painful to post...but I got it! Thanks for ur patience!

Reply:
That is not a C enum.

Reply:
Ah, so I should just declare enum exactly as you did without any type of format sanitizers like ` or ```?

E.g.
```enum eFollowPointRouteMode {
	TICKET_SINGLE = 0,
	TICKET_RETURN = 1,
	TICKET_SEASON = 2,
	TICKET_LOOP = 3
}```

I'm tryin' here! LOL seriously, thanks 4 ur patience

Reply:
Made the changes.  Let me know what you think